data_science/terraform/notebooks/iam.tf

```diff
@@ -1,3 +1,38 @@
+data "aws_iam_policy_document" "data_science_bucket" {
+  statement {
+    actions = [
+      "s3:*",
+    ]
+
+    resources = [
+      "${var.s3_bucket_arn}",
+      "${var.s3_bucket_arn}/*",
+    ]
+  }
+}
+
+data "aws_iam_policy_document" "data_science_readall" {
+  statement {
+    actions = [
+      "s3:Get*",
+    ]
+
+    resources = [
+      "*",
+    ]
+  }
+
+  statement {
+    actions = [
+      "s3:ListAllMyBuckets",
+    ]
+
+    resources = [
+      "arn:aws:s3:::*",
+    ]
+  }
+}
+
```

Comment:
Should we restrict these permissions a bit more?

Reply:
Perhaps. My reading of the permissions here is that @harrisonpim can do anything to the data science bucket, and read everything else which seems ok to me?